### PR TITLE
chore(updates): update the community pages per SIG discussion

### PR DESCRIPTION
**Pull request by 317brian**
This PR does the following:
- removes stack overflow
- removes the forum
- directs people to slack first and github second

PR for community repo: https://github.com/spinnaker/community/pull/76

### CODE REVIEW

**Comment by 317brian** [comment]
@gcomstock Here's a PR for the changes we discussed in the SIG Meeting. I kind of want to remove the support policy page and integrate it with the main page since a lot of the information is duplicated. But that's something we can think about later.

File: community/index.md [hunk 1]
```diff
@@ -168,24 +168,39 @@ chime_user:
...
 
-The Spinnaker OSS project was started at Netflix, and today thrives on the contributions and broad adoption of the public devops community. Here are some ways to engage the community and find help&mdash;join us!
+Here are some ways to engage the Spinnaker community and find help&mdash;join us!
+
+### [Slack](http://join.spinnaker.io)
 
-* [Spinnaker community forum](https://community.spinnaker.io)
+Start here! Many Spinnaker contributors and users are active on the Spinnaker Slack. It's a great place to get answers to questions or start a discussion about a feature or topic. Good places to begin on Slack include the following channels: 
+
+* A SIG channel if there is a relevant one, such as `#sig-security`. All SIG slack channels start with the prefix `sig`.
```
Comment (gcomstock) [inline community/index.md]: We should include a brief description of what a SIG is before this

**Comment by gcomstock** [comment]
Sorry for my tardiness here! Swamped with other work. Looks good w/my one comment.

File: community/index.md (hunk 1 above)
Comment (317brian) [reply community/index.md]: Added

**Comment by 317brian** [comment]
Huh, it does not like #slack because Jekyll thinks that's an internal anchor link. Will do some cleanup.

**Pull request merged**